backend/src/main/scala/com/softwaremill/adopttapir/starter/content/DirectoryMerger.scala

```diff
@@ -0,0 +1,48 @@
+package com.softwaremill.adopttapir.starter.content
+
+object DirectoryMerger {
+
+  def createTree(topDirName: String, pathNames: List[String], content: String): Directory = {
```

Comment:
I would insist that `createTree` does not belong in a `DirectoryMerger` object :) Instead, it looks like a separate `DirectoryCreator.apply` object